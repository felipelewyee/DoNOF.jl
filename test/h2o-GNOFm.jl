using DoNOF

mol = """
0 1
 O  0.0000   0.000   0.121
 H  0.0000   0.751  -0.485
 H  0.0000  -0.751  -0.485
"""

bset,p = DoNOF.molecule(mol,"cc-pvdz",spherical=true)

p.ipnof = 9

DoNOF.energy(bset,p)

