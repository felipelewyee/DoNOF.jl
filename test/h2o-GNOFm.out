Computing Integrals
Elapsed time:    1.73 Seconds

Number of basis functions                   (NBF)    = 24
Number of auxiliary basis functions         (NBFAUX) = 116
Inactive Doubly occupied orbitals up to     (NO1)    = 0
No. considered Strongly Doubly occupied MOs (NDOC)   = 5
No. considered Strongly Singly occupied MOs (NSOC)   = 0
NO. of Weakly occ. per St. Doubly occ.  MOs (NCWO)   = 3
Dimension of the Nat. Orb. subspace         (NBF5)   = 20
No. of electrons                                     = 10
    No. of alpha electrons                           = 5
    No. of beta electrons                            = 5
Multiplicity                                         = 1

Geometry
========
 O       0.0000000       0.0000000       0.1210000
 H       0.0000000       0.7510000      -0.4850000
 H       0.0000000      -0.7510000      -0.4850000

No violations of the orthonormality
No violations of the orthonormality
 
PNOF9/ADABelief Calculation
==================
 
  Nitext  Nit_orb  Time   Nit_occ  Time         Eelec          Etot           Ediff      maxdiff    MaxGrad_orb  Grad_occ
     1      10    8.9e+00    7    1.1e+01   -83.43914670   -74.31298422 -10082.43914670   0.383598   1.5e+00   1.5e-04 0.00
     2      10    2.0e-02   25    1.3e-03   -84.92771659   -75.80155411    -1.48856989   0.143452   5.7e-01   1.4e-04 0.01
     3      10    2.3e-03   14    6.2e-04   -85.31364920   -76.18748673    -0.38593262   0.056864   2.3e-01   1.9e-04 0.02
     4      10    8.5e-03    9    5.2e-04   -85.36188211   -76.23571963    -0.04823291   0.019730   7.9e-02   1.8e-04 0.02
     5      10    1.8e-03    5    3.5e-04   -85.36554045   -76.23937797    -0.00365834   0.020048   8.0e-02   1.3e-04 0.02
     6      10    4.0e-03    7    4.4e-04   -85.36748599   -76.24132351    -0.00194554   0.016372   6.5e-02   1.5e-04 0.02
     7      10    1.8e-03    7    4.2e-04   -85.36869335   -76.24253087    -0.00120736   0.018720   7.5e-02   1.3e-04 0.03
     8      10    3.3e-03    7    4.4e-04   -85.36971725   -76.24355477    -0.00102390   0.019848   7.9e-02   8.0e-05 0.03
     9      10    3.2e-03    4    3.1e-04   -85.37044408   -76.24428160    -0.00072683   0.020477   8.2e-02   1.4e-04 0.03
    10      10    1.9e-03    7    4.2e-04   -85.37075996   -76.24459749    -0.00031588   0.021135   8.5e-02   1.0e-04 0.03
    11      10    1.9e-03    3    2.6e-04   -85.37082670   -76.24466423    -0.00006674   0.020015   8.0e-02   1.4e-04 0.03
    12      10    3.2e-03    3    2.7e-04   -85.37098700   -76.24482452    -0.00016029   0.018999   7.6e-02   1.5e-04 0.03
    13      10    1.9e-03    6    4.1e-04   -85.37164842   -76.24548594    -0.00066142   0.019811   7.9e-02   1.4e-04 0.03
    14      10    1.8e-03    6    3.7e-04   -85.37334746   -76.24718499    -0.00169904   0.020809   8.3e-02   1.5e-04 0.03
    15      10    3.2e-03    4    2.9e-04   -85.37526840   -76.24910593    -0.00192094   0.017393   7.0e-02   1.5e-04 0.03
    16      10    1.9e-03    4    1.6e-03   -85.37641584   -76.25025336    -0.00114743   0.020452   8.2e-02   1.7e-04 0.03
    17      10    1.8e-03    3    2.7e-04   -85.37681346   -76.25065098    -0.00039762   0.022239   8.9e-02   1.8e-04 0.03
    18      10    3.2e-03    3    3.0e-04   -85.37712727   -76.25096480    -0.00031382   0.020829   8.3e-02   1.3e-04 0.03
    19      10    3.2e-03    3    2.9e-04   -85.37727523   -76.25111276    -0.00014796   0.019474   7.8e-02   1.4e-04 0.03
    20      10    1.8e-03    1    2.1e-04   -85.37738998   -76.25122751    -0.00011475   0.021528   8.6e-02   1.6e-04 0.03
    21      10    3.1e-03    3    2.6e-04   -85.37748724   -76.25132477    -0.00009726   0.018714   7.5e-02   9.9e-05 0.03
    22      10    3.1e-03    1    2.2e-04   -85.37753941   -76.25137693    -0.00005217   0.022085   8.8e-02   1.0e-04 0.03
    23      10    1.8e-03    0    2.0e-04   -85.37761099   -76.25144852    -0.00007158   0.019323   7.7e-02   1.3e-04 0.03
    24      20    4.8e-03    0    1.9e-04   -85.37811600   -76.25195353    -0.00050501   0.000622   2.5e-03   1.4e-04 0.03
 
RESULTS OF THE OCCUPATION OPTIMIZATION
========================================
   1    1.9999669  -20.57284362
   2    1.9855945  -1.30293952
   3    1.9707998  -0.70206483
   4    1.9707188  -0.70164119
   5    1.9779052  -0.53061842
   6    0.0151515  -0.01837074
   7    0.0247471  -0.03265656
   8    0.0249410  -0.03280577
   9    0.0072875  -0.01312066
  10    0.0000142  -0.00026873
  11    0.0058632  -0.00931837
  12    0.0037982  -0.00734052
  13    0.0038339  -0.00740071
  14    0.0037031  -0.00833517
  15    0.0000115  -0.00022127
  16    0.0010801  -0.00196420
  17    0.0007359  -0.00152994
  18    0.0004252  -0.00102375
  19    0.0034149  -0.00798231
  20    0.0000073  -0.00015854
 
----------------
 Final Energies 
----------------
Final NOF Total Energy =     -76.2519535
 
Total Orbital Internal Iterations = 250
Total Occupation Internal Iterations = 132
 
Elapsed time:   26.49 Seconds

---------------------------------
   Lowdin Population Analysis    
---------------------------------
 Idx  Atom   Population   Charge 
---------------------------------
  1     O     8.462      -0.462
  2     H     0.769       0.231
  3     H     0.768       0.232

---------------------------------
   M Diagnostic: 0.03
---------------------------------

